CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Add release workflows.
+
+### Changed
+
+- Update `go.mod` file.
+
```

Comment:
I'd say none of that is relevant to the end user and should be removed from changelog. Is updating `go.mod` a change that causes new major? 

https://github.com/giantswarm/fmt/blob/master/releases/changelog.md

Reply:
Don't know how we should handle version changes on dependency updates. You pretty much rely on anyone else to adhere to the versioning best practices but that does not guarantee anything. 

In my case here, as a user, I would have wanted the information that dependencies got updated in the changelog. Should maybe have been more detailed then with a good reason.

Reply:
I don't think that matters for the library. After all you decide on dependencies in the code using the library.